report-generator/src/main/kotlin/com/teamscale/report/util/FileSystemUtils.kt

```diff
@@ -0,0 +1,32 @@
+package com.teamscale.report.util
+
+import java.io.File
+import java.io.IOException
+
+object FileSystemUtils {
+
+    /** Unix file path separator  */
+    val UNIX_SEPARATOR = '/'
+
+    /**
+     * Replace platform dependent separator char with forward slashes to create
+     * system-independent paths.
+     */
+    fun normalizeSeparators(path: String): String {
```

Comment:
please reuse this. I've seen at least one other place where this code is cloned

Reply:
The same logic is used in javaws, but I didn't wanted to add the whole report-generator + jacoco-runtime+teamscale-client dependency tree just to share this single line of code.